packages/tooltip/src/index.js

```diff
@@ -262,6 +261,25 @@ export function useTooltip({
 
   useEffect(() => checkStyles("tooltip"));
 
+  useEffect(() => {
+    const handleKeyDown = event => {
+      if (
+        event.key === "Enter" ||
+        event.key === " " ||
+        event.key === "Escape"
```

Comment:
I think this approach is probably OK for the escape key, but not for `Enter` or `Spacebar`. I'd probably define a separate handler inside this `useEffect` hook and attach it to document (exactly as you are doing), but leave the current handler in place (keeping it wrapped along with the user-provided `onKeyDown`.

I'll still need to test this and run it by some a11y folks since keyboard language in the WIA-ARIA spec is a bit vague here, but we'll need the above changed at minimum.

Reply:
In that case, would it make sense to add/remove the event listener on mouseEnter/mouseLeave instead of useEffect so that keyboard users aren't affected? Does this handler also need to be preventable?